### PR TITLE
games-action/minetest_game: version bump to 5.0.1

### PR DESCRIPTION
**Pull request by vilhelmgray**
Signed-off-by: William Breathitt Gray <vilhelm.gray@gmail.com>

### CODE REVIEW

File: games-action/minetest_game/minetest_game-5.0.1.ebuild [hunk 1]
```diff
@@ -0,0 +1,25 @@
+# Copyright 1999-2019 Gentoo Foundation
```
Comment (juippis) [inline games-action/minetest_game/minetest_game-5.0.1.ebuild]: Why don't you commit with repoman to update this automatically to its correct form :)

File: games-action/minetest_game/minetest_game-5.0.1.ebuild [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
```
Comment (juippis) [inline games-action/minetest_game/minetest_game-5.0.1.ebuild]: 7 should work here.

File: games-action/minetest_game/minetest_game-5.0.1.ebuild [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+
+inherit vcs-snapshot
```
Comment (juippis) [inline games-action/minetest_game/minetest_game-5.0.1.ebuild]: Do you really need this?

File: games-action/minetest_game/minetest_game-5.0.1.ebuild [hunk 4]
```diff
@@ -0,0 +1,23 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+DESCRIPTION="The main game for the Minetest game engine"
+HOMEPAGE="https://github.com/minetest/minetest_game"
+SRC_URI="https://github.com/minetest/${PN}/archive/${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="GPL-2 CC-BY-SA-3.0"
```
Comment (mgorny) [inline games-action/minetest_game/minetest_game-5.0.1.ebuild]: I wanted to say this is such a nice simple ebuild but actually, it isn't. The files apparently have more licenses than you've noted here. Please look through license information in all the installed files, and assemble a complete list.

**Comment by mgorny** [comment]
Thanks.

**Pull request closed without merging**